### ISSUE TITLE
System.PlatformNotSupportedException: Operation is not supported on this platform.

### ISSUE THREAD

**Issue by mkopec3** [PowerShell/PSReadLine#1156]
Environment data
----------------

PS version: 6.2.3
PSReadline version: 2.0.0-beta3
os: Linux pearl 5.3.0-kali1-amd64 #1 SMP Debian 5.3.7-1kali2 (2019-11-04) x86_64 GNU/Linux
PS file version: 6.2.3.0
HostName: ConsoleHost
BufferWidth: 104
BufferHeight: 55

Steps to reproduce or exception report
--------------------------------------

Last 200 Keys:
```
 Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace T e Tab Bac
 UpArrow Enter
 UpArrow Space / t m p Tab t e s Tab Enter
 UpArrow Enter
 UpArrow a Enter
 UpArrow Enter
 UpArrow Enter
 UpArrow Enter
 r m r Space / t m Tab t e r Backspace s Tab Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace D e l Tab e Tab Tab Tab Backspace Backspace Backspace Backspace Backspace Backspace Backspace r e m o Tab Tab Tab v e Tab - I Tab Backspace Backspace I Tab Space / t m Tab t e Tab Enter
 UpArrow Enter
 UpArrow UpArrow Enter
 R e m o Tab M o Tab Space S h Tab Enter
 I m p Tab - M o Tab Space . \ S h Tab Space - V e r Tab Enter
 UpArrow Backspace Backspace Backspace Backspace b o Tab Enter
 G e t - V e Tab Enter
 PageDown
```

Exception:
```
System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.ConsolePal.SetWindowPosition(Int32 left, Int32 top)
   at System.Console.SetWindowPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.SetWindowPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ScrollDisplayDown(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

**Comment by msftrncs** [comment]
This is fixed in PSReadLine 2.0.0-beta5 available from PowerShellGallery.

The operation `PageDown` (and others) is not supported on non-windows environments, and so beta 5 does not install the keyhandler on non-windows systems.    You can also remove the keyhandler(s) with `Remove-PSReadLineKeyHandler PageDown`.

**Comment by mkopec3** [comment]
It seems that PSReadLine update solves the issue. Thanks for your suggestions!
